src/fulltrade/TradesContainer.js

```diff
@@ -15,8 +15,11 @@ export default class TradesContainer extends React.Component {
     };
     createTrade() {
         const { trades, actions } = this.props;
```

Comment:
@borisyankov here is the button to create additional trade panel ~

The layout probably should be in this component ~ I've added some logic here, if we want to keep container as pure data layer, we can refactor